Lib/enum.py

```diff
@@ -568,10 +568,11 @@ def __new__(metacls, cls, bases, classdict, *, boundary=None, _simple=False, **k
         try:
             exc = None
             enum_class = super().__new__(metacls, cls, bases, classdict, **kwds)
-        except RuntimeError as e:
-            # any exceptions raised by member.__new__ will get converted to a
-            # RuntimeError, so get that original exception back and raise it instead
-            exc = e.__cause__ or e
+        except Exception as e:
```

Comment:
Exception can catch any type of exception, and I'm not sure if other exceptions need to be caught here.

Reply:
Every exception should be caught so the added text can be removed.